Source/Scene/Cesium3DTileGroundPrimitive.js

```diff
@@ -0,0 +1,875 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/Cartographic',
+        '../Core/Color',
+        '../Core/ColorGeometryInstanceAttribute',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/Ellipsoid',
+        '../Core/Geometry',
+        '../Core/GeometryAttribute',
+        '../Core/GeometryAttributes',
+        '../Core/GeometryInstance',
+        '../Core/IndexDatatype',
+        '../Core/Matrix4',
+        '../Core/OrientedBoundingBox',
+        '../Core/PrimitiveType',
+        '../Core/Rectangle',
+        '../Core/TranslationRotationScale',
+        '../Renderer/Buffer',
+        '../Renderer/BufferUsage',
+        '../Renderer/DrawCommand',
+        '../Renderer/RenderState',
+        '../Renderer/ShaderProgram',
+        '../Renderer/ShaderSource',
+        '../Renderer/VertexArray',
+        '../Shaders/ShadowVolumeFS',
+        '../Shaders/ShadowVolumeVS',
+        './BlendingState',
+        './DepthFunction',
+        './Pass',
+        './StencilFunction',
+        './StencilOperation'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        Color,
+        ColorGeometryInstanceAttribute,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        destroyObject,
+        Ellipsoid,
+        Geometry,
+        GeometryAttribute,
+        GeometryAttributes,
+        GeometryInstance,
+        IndexDatatype,
+        Matrix4,
+        OrientedBoundingBox,
+        PrimitiveType,
+        Rectangle,
+        TranslationRotationScale,
+        Buffer,
+        BufferUsage,
+        DrawCommand,
+        RenderState,
+        ShaderProgram,
+        ShaderSource,
+        VertexArray,
+        ShadowVolumeFS,
+        ShadowVolumeVS,
+        BlendingState,
+        DepthFunction,
+        Pass,
+        StencilFunction,
+        StencilOperation) {
+    'use strict';
+
+    function Cesium3DTileGroundPrimitive(options) {
```

Comment:
>Is this tightly coupled to 3D Tiles?

No. As long as we can get the same information from another vector format, we can use this.